packages/peregrine/lib/talons/CartPage/PriceSummary/usePriceSummary.js

```diff
@@ -25,10 +25,7 @@ export const usePriceSummary = props => {
     const [{ cartId }] = useCartContext();
 
     const [fetchPriceSummary, { error, loading, data }] = useLazyQuery(
-        props.query,
-        {
-            fetchPolicy: 'no-cache'
```

Comment:
You removed this cause refetch refreshes this data right?

Reply:
Correct. Since I didn't know where to track the remaining work for making sure aggressive fetch policies are removed on completion, I just removed them entirely. You'll need to delete the apollo-cache entry in local storage and refresh, and you should see live data.

Reply:
This is on a fresh start, cleared cache and everything.

If you start from the `/cart` page I believe it never is updated after future operations.

Reply:
After discussion we have opened [a ticket](https://jira.corp.magento.com/browse/PWA-321) to track the ideal state work.